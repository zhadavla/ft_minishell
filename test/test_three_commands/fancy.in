echo "Hello, world!" | tr '[:lower:]' '[:upper:]'
find /usr/bin -name "zip" -type f -print0 | xargs -0 ls -l
cat /etc/passwd | cut -d: -f1 | sort | uniq | wc -l
date -d "2022-01-01" +%A
