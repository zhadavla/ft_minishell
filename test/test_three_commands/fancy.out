minishell$ echo "Hello, world!" | tr '[:lower:]' '[:upper:]'
HELLO, WORLD!
minishell$ find /usr/bin -name "zip" -type f -print0 | xargs -0 ls -l
-rwxr-xr-x 1 root root 203768 Mar 25  2022 /usr/bin/zip
minishell$ cat /etc/passwd | cut -d: -f1 | sort | uniq | wc -l
53
minishell$ date -d "2022-01-01" +%A
Saturday
minishell$ exit
